granule.go

```diff
@@ -211,5 +238,130 @@ func (g *Granule) Less(than btree.Item) bool {
 
 // Least returns the least row in a Granule.
 func (g *Granule) Least() *dynparquet.DynamicRow {
-	return (*dynparquet.DynamicRow)(g.least.Load())
+	return (*dynparquet.DynamicRow)(g.metadata.least.Load())
+}
+
+// minmaxes finds the mins and maxes of every column in a part.
+func (g *Granule) minmaxes(p *Part) error {
+	f := p.Buf.ParquetFile()
+	numRowGroups := f.NumRowGroups()
```

Comment:
The Parquet ColumnIndex seems like it already gets us most of the way here, no?

Reply:
Hmm I can't figure out how to actually extract all the correct values. I see the label values, but none of the non string values.

Reply:
Only problem I'm running into using the column index is we're truncating values. 
ex: `process_cpu_cpu_nanoseconds` is being stored as the index max as `process_cpu_cpu`

Reply:
Looks like it's a writer setting that we control. So I've changed it to use that setting to truncate during comparison.